2020-01-Winter/0_arrays_sorting/README.md

```diff
@@ -118,18 +136,53 @@ Go to [Top](#top)
 
 ### 2. SOLUTION 2 TODO :bug:
 
-Source: TODO :bug:
+Source: <a href="https://leetcode.com/problems/search-in-rotated-sorted-array/">LeetCode</a> :bug:
 
 #### Naive/Simple Solution
 
 TODO :bug:
 
-#### Optimal Solution
+#### Optimal Solution O(log n)
+
```

Comment:
Please use Markdown here so the text is monospaced and has syntax highlighting.

````suggestion

```c++
````